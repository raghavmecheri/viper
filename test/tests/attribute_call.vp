int func tester(int x) {
  return 42;
}

bool func least(int a, int b, int d, int e, int f) {
  return a > b;
}

int z = 42.tester();
print("testing with no parameters:");
print(z);

print("testing with many parameters:");
int isLeast = 5;
print(isLeast.least(6, 3, -9, 88));