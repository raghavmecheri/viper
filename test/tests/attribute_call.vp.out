testing with no parameters:
42
testing with many parameters:
0